### PR TITLE
Fixed bug in which adding new map marker crashed app for all current …

### PR DESCRIPTION
**Pull request by adebroux**
…users

### CODE REVIEW

**Comment by adebroux** [comment]
[App crashes when new markers appear on map(home page)](https://trello.com/c/Cbde4w86/40-app-crashes-when-new-markers-appear-on-maphome-page)

**Review by scott-massey — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@protonapp/map-component",
-  "version": "0.1.6",
+  "version": "0.1.7",
```
Comment (scott-massey) [inline package.json:3]:
```suggestion
  "version": "0.1.8",
```

File: src/Map/map.js [hunk 2]
```diff
@@ -91,9 +91,9 @@ export const getMap = (
           <Marker
             coordinate={{
               latitude:
-                addresses.length > 0 ? addresses[index].location.lat : 0,
+                addresses.length > 0 ? addresses[index] && addresses[index].location.lat : 0,
```
Comment (scott-massey) [inline src/Map/map.js:94]:
```suggestion
                addresses.length > 0 && addresses[index] ? addresses[index].location.lat : 0,
```
Let's try to keep all of the testing logic in the first part of the ternary, keeps things cleaner.
And make sure to make the same change for the other lines too.

**Review by scott-massey — COMMENTED** [review]
Just one small change. Awesome job finding `getDerivedStateFromProps`!

**Review by scott-massey — APPROVED** [review]

**Pull request merged**